src/nvim/ex_cmds.c

```diff
@@ -64,9 +65,14 @@
  */
 typedef struct sign sign_T;
 
+// boolean to know if we have to undo
+static int EVENT_SUB = 0;
+static int sub_done = 0;
 
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "ex_cmds.c.generated.h"
+#include "ex_cmds.h"
```

Comment:
This should be moved to the other includes before the `#ifdef`. Also the `nvim/` prefix is missing.